trackman/templates/library/artist.html

```diff
@@ -14,6 +14,35 @@
 <h1>Artist: {{ artist }}</h1>
 
 {{ macros.library_tracklist(tracks, edit_from='artist') }}
+
+<div class="card">
+    <div class="card-header">
+        MusicBrainz Release Groups
+    </div>
+    <div class="card-body">
+        <form action="{{ url_for('trackman_library.artist_musicbrainz') }}" method="get" class="form-horizontal" role="form">
+            <input type="hidden" name="artist" value="{{ artist }}"/>
+
+            <div class="form-group row">
+                <label for="artist_mbid" class="col-sm-2 control-label col-form-label">Artist MBID</label>
+                <div class="col-sm">
+                    <input class="form-control" type="text" name="artist_mbid" id="artist_mbid" list="artist_mbids" autocomplete="off"/>
+                    <datalist id="artist_mbids">
+{% for artist_mbid in artist_mbids %}
```

Comment:
not really groking why we're iterating here. is this for cases where there are multiple release artists?

Reply:
Yes, we take in a list of every artist MBID seen for every Track on the artist page. In cases where there are multiple artists with the same name, this could be a list rather than a single entry.

Reply:
okay. this was subtle in testing the UI, so I didn't understand where it came from. The UI here is clunky, but I can live with it.